src/tests/unit/tests/DetailsView/components/tab-stops-failed-instance-section.test.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { VisualizationScanResultData } from 'common/types/store-data/visualization-scan-result-data';
+import {
+    TabStopsFailedInstanceSection,
+    TabStopsFailedInstanceSectionDeps,
+    TabStopsFailedInstanceSectionProps,
+} from 'DetailsView/components/tab-stops-failed-instance-section';
+import { TabStopsFailedCounter } from 'DetailsView/tab-stops-failed-counter';
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { IMock, Mock } from 'typemoq';
+
+describe('TabStopsFailedInstanceSection', () => {
+    let tabStopsFailedCounterMock: IMock<TabStopsFailedCounter>;
+
+    const visualizationScanResultDataStub = {
+        tabStops: { requirements: {} },
+    } as VisualizationScanResultData;
+
+    let props = {} as TabStopsFailedInstanceSectionProps;
+    let depsStub = {} as TabStopsFailedInstanceSectionDeps;
+
+    beforeAll(() => {
+        tabStopsFailedCounterMock = Mock.ofType(TabStopsFailedCounter);
```

Comment:
You're missing the validation of the interaction with this mock.

On another note: quiet surprisingly the badgeCount property is ... just not there in the snapshot? An odd jest thing to do ...